pkg/controller/operator/seed/reconciler_test.go

```diff
@@ -279,6 +293,64 @@ func TestBasicReconciling(t *testing.T) {
 				return nil
 			},
 		},
+
+		{
+			name:            "nodeport-proxy annotations are carried over to the loadbalancer service",
+			seedToReconcile: "seed-with-nodeport-proxy-annotations",
+			configuration: &operatorv1alpha1.KubermaticConfiguration{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test",
+					Namespace: "kubermatic",
+				},
+				Spec: operatorv1alpha1.KubermaticConfigurationSpec{
+					Ingress: operatorv1alpha1.KubermaticIngressConfiguration{
+						Domain: "example.com",
+					},
+				},
+			},
+			seedsOnMaster: []string{"seed-with-nodeport-proxy-annotations"},
+			syncedSeeds:   sets.NewString("seed-with-nodeport-proxy-annotations"),
+			assertion: func(test *testcase, reconciler *Reconciler) error {
+				if err := reconciler.reconcile(reconciler.log, test.seedToReconcile); err != nil {
+					return fmt.Errorf("reconciliation failed: %v", err)
+				}
+
+				seedClient := reconciler.seedClients["seed-with-nodeport-proxy-annotations"]
+
+				seed := kubermaticv1.Seed{}
+				if err := seedClient.Get(reconciler.ctx, types.NamespacedName{
+					Namespace: "kubermatic",
+					Name:      "seed-with-nodeport-proxy-annotations",
+				}, &seed); err != nil {
+					return fmt.Errorf("failed to retrieve Seed: %v", err)
+				}
+
+				if !kubernetes.HasFinalizer(&seed, common.CleanupFinalizer) {
+					return fmt.Errorf("Seed copy in seed cluster does not have cleanup finalizer %q", common.CleanupFinalizer)
+				}
+
+				svc := corev1.Service{}
+				if err := seedClient.Get(reconciler.ctx, types.NamespacedName{
+					Namespace: "kubermatic",
+					Name:      "nodeport-proxy",
+				}, &svc); err != nil {
+					return fmt.Errorf("failed to retrieve nodeport-proxy Service: %v", err)
+				}
+
+				if svc.Annotations == nil {
+					return fmt.Errorf("Nodeport service in seed cluster does not have configured annotations: %q", allSeeds["seed-with-nodeport-proxy-annotations"].Spec.NodeportProxy.Annotations)
+				}
+
+				for k, v := range allSeeds["seed-with-nodeport-proxy-annotations"].Spec.NodeportProxy.Annotations {
+					fmt.Printf("svc.Annotations[k]: %s, k: %s, v: %s", svc.Annotations[k], k, v)
```

Comment:
Please don't print random things during tests. :)

Reply:
oops. that was not supposed to stay there, thanks!